src/main/java/com/hyperlocal/server/Data/ShopDetails.java

```diff
@@ -0,0 +1,57 @@
+package com.hyperlocal.server.Data;
+
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class ShopDetails {
+    public Shop shop;
+    public Merchant merchant;
+    public List<CatalogItem> catalog;
+
+    public ShopDetails()
+    {
+        this.shop = null;
+        this.merchant = null;
+        this.catalog = new ArrayList<CatalogItem>();
+    }
+
+    public ShopDetails(Shop shop)
+    {
+        this.shop = shop;
+        this.merchant = null;
+        this.catalog = new ArrayList<CatalogItem>();
+    }
+
+    public ShopDetails(Shop shop, Merchant merchant, List<CatalogItem> catalog)
+    {
+        this.shop = shop;
+        this.merchant = merchant;
+        this.catalog = catalog;
+    }
+
+    public void updateShop(Shop shop)
```

Comment:
we generally use set.* terminology. 

Would be better to rename these as setShop and setMerchant.